app/run.py

```diff
@@ -73,113 +73,115 @@ def generate_parser(parser=None):
+             'found under the BIDS input directory. The participant label '
+             'does not include the "sub-" prefix'
     )
     parser.add_argument(
         '--freesurfer-license', dest='freesurfer_license', 
         metavar='LICENSE_FILE',
-        help='if using docker or singularity, you will need to acquire and '
-             'provide your own freesurfer license. The license can be '
+        help='If using docker or singularity, you will need to acquire and '
+             'provide your own FreeSurfer license. The license can be '
              'acquired by filling out this form: '
              'https://surfer.nmr.mgh.harvard.edu/registration.html'
     )
     parser.add_argument(
         '--all-sessions', dest='collect', action='store_true',
-        help='collapses all sessions into one when running a subject.'
+        help='Collapses all sessions into one when running a subject.'
     )
     parser.add_argument(
         '--ncpus', type=int, default=1,
-        help='number of cores to use for concurrent processing and '
+        help='Number of cores to use for concurrent processing and '
              'algorithmic speedups.  Warning: causes ANTs and FreeSurfer to '
              'produce non-deterministic results.'
     )
     parser.add_argument(
         '--stage',
-        help='begin from a given stage, continuing through.  Options: '
+        help='Begin from a given stage, continuing through.  Options: '
              'PreFreeSurfer, FreeSurfer, PostFreeSurfer, FMRIVolume, '
              'FMRISurface, DCANBOLDProcessing, ExecutiveSummary, CustomClean'
     )
     parser.add_argument(
         '--bandstop', type=float, nargs=2, metavar=('LOWER', 'UPPER'),
-        help='parameters for motion regressor band-stop filter. It is '
+        help='Parameters for motion regressor band-stop filter. It is '
              'recommended for the boundaries to match the inter-quartile '
-             'range for participant group respiratory rate (bpm), or to match '
-             'bids physio data directly [3].  These parameters are highly '
-             'recommended for data acquired with a frequency of approx. 1 Hz '
-             'or more (TR<=1.0). Default is no filter'
+             'range for participant group respiratory rate (breaths per '
+             'minute), or to match bids physio data directly [3].  These '
+             'parameters are highly recommended for data acquired with a '
+             'frequency of approx. 1 Hz or more (TR<=1.0). Default is no '
+             'filter.'
     )
     extras = parser.add_argument_group(
-        'special pipeline options',
-        description='options which pertain to an alternative pipeline or an '
-                    'extra stage which is not\n inferred from the bids data.'
+        'Special pipeline options',
+        description='Options which pertain to an alternative pipeline or an '
+                    'extra stage which is not\n inferred from the BIDS data.'
```

Comment:
Do you mean to have the \n after not?

Reply:
Yes.  argparse "descriptions" are formatted raw, unlike the help text.  It was to get the citations at the end to format properly.